compiler/rustc_middle/src/dep_graph/mod.rs

```diff
@@ -94,7 +94,7 @@ impl<'tcx> DepContext for TyCtxt<'tcx> {
     }
 
     #[inline]
-    fn dep_kind_info(&self, dep_kind: DepKind) -> &DepKindStruct<'tcx> {
-        &self.query_kinds[dep_kind as usize]
+    fn dep_kind_info(&self, dk: DepKind) -> &DepKindStruct<'tcx> {
+        &self.query_kinds[dk as usize]
```

Comment:
Why was this necessary?

Reply:
There is a `DepKind::dep_kind` variant. The former version did not handle hygiene correctly, so did not fire in case of macro expansion. The new version just ignores hygiene, so fires. (In that case, it's actually the more correct behaviour.)